bdk_cbf_example/src/main.rs

```diff
@@ -0,0 +1,158 @@
+use bdk_cli::{
+    anyhow,
+    clap::{self, Args, Subcommand, ValueEnum},
+};
+
+use crate::cbf::CbfEvent;
+use bdk_chain::chain_graph::ChainGraph;
+use bdk_chain::keychain::{DerivationAdditions, KeychainChangeSet};
+use bdk_chain::{BlockId, TxHeight};
+
+mod cbf;
+
+#[derive(Args, Debug, Clone)]
+struct CbfArgs {
+    #[arg(value_enum, default_value_t = Domains::IPv4)]
+    domains: Domains,
+}
```

Comment:
I'd prefer it if `domains` were just arguments to the actual commands that use `domains` and to not have `CbfArgs`. FWIW I've complained about adding global arguments to the `bdk_cli::init` to @evanlinjin but it looks like he didn't remove it from that PR yet.